hpo_demo_setup.sh

```diff
@@ -15,6 +15,10 @@
 # limitations under the License.
```

Comment:
Can you please update this file to have tabs uniformly, currently it is bad mix of both

Reply:
done